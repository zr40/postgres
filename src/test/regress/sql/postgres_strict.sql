SET postgres_strict.disable = 'none';

--
-- require_column_as
--

SELECT 1 AS a;
SELECT 1 a;
SELECT 1a;

DO $$
DECLARE
_r record;
BEGIN
SELECT 1 a INTO _r;
END $$;

DO $$
BEGIN
RAISE NOTICE '%', 1 a;
END $$;
