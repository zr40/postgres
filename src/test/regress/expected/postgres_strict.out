SET postgres_strict.disable = 'none';
--
-- require_column_as
--
SELECT 1 AS a;
 a 
---
 1
(1 row)

SELECT 1 a;
ERROR:  AS omitted for column alias
LINE 1: SELECT 1 a;
                 ^
SELECT 1a;
ERROR:  AS omitted for column alias
LINE 1: SELECT 1a;
                ^
DO $$
DECLARE
_r record;
BEGIN
SELECT 1 a INTO _r;
END $$;
ERROR:  AS omitted for column alias
LINE 5: SELECT 1 a INTO _r;
                 ^
DO $$
BEGIN
RAISE NOTICE '%', 1 a;
END $$;
ERROR:  AS omitted for column alias
LINE 3: RAISE NOTICE '%', 1 a;
                            ^
